CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.47.0] - 2023-11-13
+- Use Node(Map) instead of D2node(Map)
```

Comment:
nit: to be more specific so people know it's about indis change, how about "Use Node and D2UriMap for observer flow"?

Reply:
Oops, yeah let me update the PR title as well